samples/MvcSubAreaSample.Web/Startup.cs

```diff
@@ -0,0 +1,78 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNet.Builder;
+using Microsoft.AspNet.Hosting;
+using Microsoft.AspNet.Mvc;
+using Microsoft.AspNet.Mvc.Razor;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace MvcSubAreaSample.Web
+{
+    public class Startup
+    {
+        public IServiceProvider ConfigureServices(IServiceCollection services)
+        {
+            services.Configure<RazorViewEngineOptions>(options =>
+            {
+                options.ViewLocationExpanders.Add(new SubAreaViewLocationExpander());
+            });
+
+            services.AddMvc(options =>
+            {
+                //options.Filters.Add(typeof(PassThroughAttribute), order: 17);
+                options.Filters.Add(new FormatFilterAttribute());
+            });
+
+            return services.BuildServiceProvider();
+        }
+
+        public IConfigurationRoot Configuration { get; set; }
+
+        // Configure is called after ConfigureServices is called.
+        public void Configure(IApplicationBuilder app)
+        {
+            // Add MVC to the request pipeline.
+            app.UseMvc(routes =>
+            {
+                routes.MapRoute("areaBase", "{area:exists}",
+                    new
+                    {
+                        controller = "Home",
+                        action = "Index"
+                    });
```

Comment:
I've condensed this down more since you asked, but for some reason Github is still showing an old version instead of marking the comment as outdated. Please look at this one under "Files Changed" to see how it's actually looking right now.